scripts/pybleau_as_chaco_api.py

```diff
@@ -0,0 +1,36 @@
+""" Illustrates how to programmatically create a DF analyzer with populated
+plots in the plotter tool.
+"""
+
+import numpy as np
+from app_common.apptools.testing_utils import wrap_chaco_plot
+from app_common.std_lib.logging_utils import initialize_logging
+from pandas import DataFrame
+
+from pybleau.app.plotting.api import plot_from_config
+
+from pybleau.app.plotting.plot_config import HeatmapPlotConfigurator, \
+    BarPlotConfigurator
+
+initialize_logging(logging_level="DEBUG")
+
+col4 = np.random.randn(8)
```

Comment:
Is this used somewhere that I'm not seeing?

Reply:
Removed.